src/components/CommentVotes.jsx

```diff
@@ -0,0 +1,15 @@
+import { Button } from "react-bootstrap";
+
+export const CommentVotes = ({ votes }) => {
+  return (
+    <div >
+      <p >Total votes: <b className="emphasise">{votes}</b></p>
+      <Button className="vote-button" variant="light">
+        Down Vote
+      </Button>{" "}
+      <Button className="vote-button" variant="dark">
+        Up Vote
+      </Button>{" "}
+    </div>
```

Comment:
The `{" "}`'s here are probably added by prettier and can be taken out unless you really want a space character between these buttons 😁